src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.io.File;
+import java.io.IOException;
+
+public class Application {
+    public static void main(String[] args) {
+        WorkWithFile workWithFile = new WorkWithFile();
+        File reportFile = new File("report.csv");
+        try {
+            reportFile.createNewFile();
+            workWithFile.getStatistic("orange.csv", "report.csv");
```

Comment:
According to the checklist, you should not hardcode file names like "orange.csv" and "report.csv". It's better to declare them as constants at the class level. This will make your code more maintainable and the purpose of these files clearer.